fallback.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "fallback to: v1.2.4"
+curl  https://codeload.github.com/strongloop/fsevents/tar.gz/v1.2.4 | tar xz
+pushd fsevents-1.2.4
+npm install node-pre-gyp
```

Comment:
I don't think it's a good idea to mess with lock files like this, it breaks too many assumptions.

IMHO it'd be better to include both the N-API and NAN versions in the repo using this strategy https://github.com/mapbox/node-pre-gyp#supporting-both-n-api-and-nan-builds. This way supporting Node >= 6; <= 8.12 would be fairly easy.

Reply:
Well, thanks to @bnoordhuis confirming that *libuv* is pretty much API/ABI stable, I changed it to that. Meaning from v6.x - v8.x we can use the same compiled binary. In other words, we don't need any of this anymore. 😄